src/content/runtime-apis/scheduled-event.md

```diff
@@ -0,0 +1,42 @@
+# ScheduledEvent
+
+## Background
+
+The event type for Scheduled requests to a Worker. The `Object` passed through as `event` in 
+[`addEventListener()`](/runtime-apis/add-event-listener).
+
+## Context
+
+```js
+addEventListener("scheduled", event => {
+  event.waitUntil(handleScheduled(event))
+})
+```
+
+### Properties
+
+<Definitions>
+
+- `event.type` <Type>string</Type>
+    - The type of event.
+
+- `event.scheduledTime` <Type>scheduledTime</Type>
+    - The time the `ScheduledEvent` was scheduled to be executed.
+
+</Definitions>
+
+### Methods
+
+When a Workers script is invoked by a [Cron Trigger](/platform/cron-triggers), the Workers runtime starts a ScheduledEvent 
```

Comment:
is `ScheduledEvent` here a constant?

Reply:
I don't understand the question.

Reply:
I see what you mean now. `ScheduledEvent`